<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-9DM6QIP" failures="3" tests="3" name="Default test" time="19.313" errors="0" timestamp="2024-11-13T22:34:58 IST">
  <testcase classname="com.avactis.testcases.RegisteredUserTest" name="RegisteredUser" time="0.114">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	The method clickOnCheckOut() is undefined for the type AddToCartPage
	The method clickOnProceedToCheckOut() is undefined for the type ShippingPage
	The method clickOnPaymentMethod() is undefined for the type PaymentPage
	The method validateConfirmMessage() is undefined for the type OrderConfirmationPage
	Assert cannot be resolved
	Log cannot be resolved
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
The method clickOnCheckOut() is undefined for the type AddToCartPage
The method clickOnProceedToCheckOut() is undefined for the type ShippingPage
The method clickOnPaymentMethod() is undefined for the type PaymentPage
The method validateConfirmMessage() is undefined for the type OrderConfirmationPage
Assert cannot be resolved
Log cannot be resolved

at com.avactis.testcases.RegisteredUserTest.RegisteredUser(RegisteredUserTest.java:56)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- RegisteredUser -->
  <testcase classname="com.avactis.testcases.RegisteredUserTest" name="RegisteredUser" time="0.005">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	The method clickOnCheckOut() is undefined for the type AddToCartPage
	The method clickOnProceedToCheckOut() is undefined for the type ShippingPage
	The method clickOnPaymentMethod() is undefined for the type PaymentPage
	The method validateConfirmMessage() is undefined for the type OrderConfirmationPage
	Assert cannot be resolved
	Log cannot be resolved
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
The method clickOnCheckOut() is undefined for the type AddToCartPage
The method clickOnProceedToCheckOut() is undefined for the type ShippingPage
The method clickOnPaymentMethod() is undefined for the type PaymentPage
The method validateConfirmMessage() is undefined for the type OrderConfirmationPage
Assert cannot be resolved
Log cannot be resolved

at com.avactis.testcases.RegisteredUserTest.RegisteredUser(RegisteredUserTest.java:56)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- RegisteredUser -->
  <testcase classname="com.avactis.testcases.RegisteredUserTest" name="RegisteredUser" time="0.008">
    <failure type="java.lang.Error" message="Unresolved compilation problems: 
	The method clickOnCheckOut() is undefined for the type AddToCartPage
	The method clickOnProceedToCheckOut() is undefined for the type ShippingPage
	The method clickOnPaymentMethod() is undefined for the type PaymentPage
	The method validateConfirmMessage() is undefined for the type OrderConfirmationPage
	Assert cannot be resolved
	Log cannot be resolved
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
The method clickOnCheckOut() is undefined for the type AddToCartPage
The method clickOnProceedToCheckOut() is undefined for the type ShippingPage
The method clickOnPaymentMethod() is undefined for the type PaymentPage
The method validateConfirmMessage() is undefined for the type OrderConfirmationPage
Assert cannot be resolved
Log cannot be resolved

at com.avactis.testcases.RegisteredUserTest.RegisteredUser(RegisteredUserTest.java:56)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- RegisteredUser -->
</testsuite> <!-- Default test -->
